main.go

```diff
@@ -207,6 +207,11 @@ func main() {
 			os.Exit(1)
 		}
 	}
+	if err = (&infrav1alpha3.AWSMachine{}).SetupWebhookWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create webhook", "webhook", "AWSMachine")
+		os.Exit(1)
+	}
+
```

Comment:
This is a duplicate of what is in the above `if webhookPort != 0`
https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/2dc2fe8e5e968b474ee1cada99609496f0bd3af0/main.go#L197-L200

Reply:
thanks, updated!